build.gradle

```diff
@@ -18,13 +18,16 @@ ext {
     cglibVersion = "3.2.0"
     objenesisVersion = "2.2"
     jaxbVersion = "2.2.11"
+    betamaxVersion = "2.0.0-alpha-1"
```

Comment:
@walro I think it's left in so we can easily switch back when a proper release of betamax has been made